android/quest/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
     signingConfigs {
         release {
 
-            v1SigningEnabled true
+            v1SigningEnabled false
```

Comment:
Why is signing false? Will this be only true when we want to publish to the playstore?

Reply:
v1 is false because of a security vulnerability on some Android versions. We should be okay with just using v2.